server/models/DeviceWearer.ts

```diff
@@ -9,7 +9,7 @@ const DeviceWearerModel = z.object({
   lastName: z.string().nullable(),
   alias: z.string().nullable(),
   dateOfBirth: z.string().datetime().nullable(),
-  adultAtTimeOfInstallation: z.boolean().nullable(),
+  adultAtTimeOfInstallation: z.coerce.boolean().nullable(),
```

Comment:
This is probably an important change to note, the wiremock response returns boolean values as a string and so zod needs to coerce strings into booleans to be successful.

Reply:
This model is only used to validate the response from the api which should only return boolean values and anything other would be incorrect. Also `.zod.coerce.boolean()` uses the `Boolean` primitive which in JS makes anything not falsy into true, e.g.

```node
> Boolean('abc')
true
```

Reply:
Created a definable response so this is not required anymore